Assets/Scripts/CameraController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraController : MonoBehaviour
+{
+    [SerializeField] List<GameObject> followingSubjectList;

```

Comment:
Private uninitialized fields give a compiler warning in the editor, consider initializing this with `followingSubjectList = default`